src/index.js

```diff
@@ -1,33 +1,133 @@
+/* eslint-disable no-loop-func */
+/* eslint-disable no-restricted-globals */
 import './styles/style.css';
 
-const todoList = [
-  {
-    discription: 'wash the dishes',
-    completed: false,
-    index: 0,
-  },
-  {
-    discription: 'complete To Do list project',
-    completed: false,
-    index: 1,
-  },
-  {
-    discription: 'implement project code review suggestions',
-    completed: false,
-    index: 2,
-  },
-];
+class TaskManager {
+  constructor(taskName) {
+    this.taskName = taskName;
+    this.completed = false;
+    this.itemIndex = 0;
+  }
+
+  addTask = (todoList) => {
+    const tasks = todoList;
+    localStorage.setItem('todos', JSON.stringify(tasks));
+    location.reload();
+  };
+
+  // remove task from localStorage && create a todoList
+  removeTask = (task) => {
+    const oldTodoList = JSON.parse(localStorage.getItem('todos'));
+    const currentTodoList = [];
+
+    // get all list todo's not deleted
+    oldTodoList.filter((item) => {
+      if (item.taskName !== task) {
+        currentTodoList.push(item);
+      }
+      return item;
+    });
+    localStorage.setItem('todos', JSON.stringify(currentTodoList));
+    location.reload();
+  };
```

Comment:
- [ ] When a task is removed the index values of the remaining items should be updated accordingly. So if I have four items with index values of 1, 2, 3, and 4 respectively, and I remove the third item, the new index values should be 1, 2, 3 - not 1, 2, 4. Kindly implement that as well.